config/environment.js

```diff
@@ -1,6 +1,20 @@
 /* eslint-env node */
 'use strict';
 
+function codeVersion() {
+  // Heroku Git Hash support
+  if (process.env.SOURCE_VERSION) {
+    let packageJson = require('../package.json');
```

Comment:
We still need to check it out on heroku. There is big chance that It will not resolve it correctly. Probably we have to move it to the build process to be able to access the app package.lock

Reply:
BTW: we should look for server provider independent solution :)

Reply:
So it is. the problem here is that Heroku does not allow you to reach the .git folder. It just saves some properties in process variables. We can just remove it but I think Heroku is one of the most popular ember app prototyping service and should be covered here, otherwise, we should document it very well in the readme.

Reply:
Yeah, it will definitely brake :/